server/Episememe.Application.Tests/FeatureTests/MediaRevision/WhenRevisingMedia.cs

```diff
@@ -0,0 +1,99 @@
+using Episememe.Application.Exceptions;
+using Episememe.Application.Features.UpdateTags;
+using Episememe.Application.Interfaces;
+using Episememe.Domain.Entities;
+using Episememe.Domain.HelperEntities;
+using Episememe.Infrastructure.Database;
+using FluentAssertions;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Episememe.Application.Tests.FeatureTests.FileRevision
+{
+    public class WhenUploadingFile : IDisposable
+    {
+        private readonly IWritableApplicationContext _contextMock;
+        private readonly DbConnection _connection;
+
+        public WhenUploadingFile()
+        {
+            (_contextMock, _connection) = GetInMemoryDatabaseContext();
+        }
+
+        [Fact]
+        public void SwapTagsInInstance()
```

Comment:
Keep the naming convention "Given_Then", e.g.: *GivenMediaTagsAreUpdated_ThenExpectedMediaTagsAreInDatabase* or something like that